src/pages/ProposalDetails/ProposalDetails.component.tsx

```diff
@@ -72,53 +72,55 @@ export const ProposalDetailsComponent = ({
         </Box>
       }
     >
-      <Grid container>
-        <Grid item sm={12} sx={{ p: ({ spacing }) => spacing(0, 2) }}>
-          <VotesRatioBlock />
-        </Grid>
+      <Box sx={{ p: ({ spacing }) => spacing(0, 2), mb: 1 }}>
+        <VotesRatioBlock />
+      </Box>
+
+      <Grid
```

Comment:
I would stay with one of 'grid' | 'flex'
![Selection_095](https://user-images.githubusercontent.com/100695408/162402727-a8db3634-21b9-4d29-a74c-a327e00e8928.png)

Reply:
> I would stay with one of 'grid' | 'flex' :P ![Selection_095](https://user-images.githubusercontent.com/100695408/162402727-a8db3634-21b9-4d29-a74c-a327e00e8928.png)

That's the approach suggested by mui
https://mui.com/system/grid/

Mui `Grid` uses flex by default, but is overriding it that bad in such situation?

Reply:
ok, it shouldn't be a problem, it is probably a fallback for older browsers :+1: